scripts/update.js

```diff
@@ -427,7 +427,7 @@ const writeChanges = (deferred) => {
   fs.writeFileSync(files.metadata, toJson(metadata))
 
   if (batch.found > 0) {
-    fs.writeFileSync(files.packages, toJson(sort(packages)))
+    fs.writeFileSync(files.packages, toJson(sort(Object.fromEntries(packages))))
```

Comment:
So you're using a Map when collecting data then turning it into an Object when finished? That makes sense to me. 👍🏼

Reply:
Yes, basically this is it. I updated the `sort` function so it will work with the Map as an input instead of an object - no need to convert the map to an object before sorting it.